index.html

```diff
@@ -0,0 +1,139 @@
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="links-container">
+    <ul>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a>Program</a>
+      </li>
+      <li>
+        <a>Join</a>
+      </li>
+      <li>
+        <a>Sponsor</a>
+      </li>
+      <li>
+        <a>News</a>
+      </li>
+      <li>
+        <button type="button">CC Campaign</button>
+      </li>
+    </ul>
+  </div>
+</nav>
+
+<header class="header">
+  <div class="header-container">
+    <h1 class="welcome-heading">"Hello! Sharing world"</h1>
+    <h2 class="intro">10-Gano Courses</h2>
+    <p class="intro-paragraph">Learn at the comfort of your own home. 10-Gano website is the world's best platform for whoever is willling to develop their skillset and advance career.</p>
+    <!-- today's date -->
+    <h3 class="date" id="date"></h3>
+  </div>
+</header>
+
+<section class="main-program">
+  <h3 class="main-program-heading">Main Program</h3>
+  <hr class="main-program-hr">
+  <ul class="programs-list">
+    <li class="program-container">
+      <img src="assets/distance-learning.png" alt="course" />
+      <h4 class="program-title">AI Marketing</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/online-test.png" alt="course" />
+      <h4 class="program-title">Fintech Programme</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/audio-book.png" alt="course" />
+      <h4 class="program-title">Cyber Security</h4>
+      <p class="program-desc">Try creating your own work using open source license</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/student.png" alt="course" />
+      <h4 class="program-title">AI Marketing</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+    <li class="program-container">
+      <img src="assets/work.png" alt="course" />
+      <h4 class="program-title">Blockchain Strategy Programme</h4>
+      <p class="program-desc">Have the time to share your thoughts and opinions with experts for each logic</p>
+    </li>
+  </ul>
+  <button type="button" class="program-join-button">Join CC Latest Course Journal</button>
+  <a href="" class="program-alt-link">see the whole program</a>
+</section>
+
+  <section class="featured-professors">
+    <h3 class="featured-professors-heading">Featured Professors</h3>
+    <hr class="main-program-hr" id="main-program-hr">
+    <article class="professors-list" id="professors-list">
+    </article>
+    <button type="button" class="more-professors-btn" id="more-professors-btn">More<img src="assets/arrow_down.png" alt="more">
+    </button>
+  </section>
```

Comment:
- [ ] Your Design 👇 

![image](https://user-images.githubusercontent.com/78251927/225742310-5d0e28da-29df-42a6-b7af-d896d4acbbb0.png)

- [ ] Template 👇 

![image](https://user-images.githubusercontent.com/78251927/225742436-7a341032-fac2-491c-8414-722df5e574a2.png)

- [ ] The Desktop version of your website is supposed to display the list of all the Featured Speakers as shown in the Template 👆 

- [ ] While the `Show More` button is only useful in the `Mobile` Design 👍Please kindly fix that 🤗

- [ ] Also let's make some adjustments in the `Layout:` (composition and space between elements) of this section. 💪